tests/ui/test_mcg_ui.py

```diff
@@ -0,0 +1,271 @@
+import logging
```

Comment:
can we put the test file under /tests/manage/mcg?
Maybe create a new hierarchy - /tests/manage/mcg/ui